nodejs-write-code.md

```diff
@@ -5,10 +5,10 @@ It is a good practice that you create a branch for each of the new feature/fixes
 
 ### 2. main.js
 your main.js should look something like below:
-```
-const open = require('./node_modules/open')
+```js
+const open = require('./node_modules/open');
```

Comment:
is adding `;` recommended for `js`?🤔 I think it's optional

Reply:
It _kinda_ is optional. Some people prefer to use them, some prefer not to use them. JavaScript has this thing called `ASI`, or `Automatic Semicolon Insertion`. Technically, not using semicolons is a syntax error, but JavaScript tries to automatically insert them whenever possible. There's a [whole list of rules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#automatic_semicolon_insertion) for this, there are some gotchas you have to always keep in mind when writing code without them, and I'm of the opinion that you shouldn't write your code with syntax errors relying on the language correcting them for you.